mandatory/1-fix-functions.js

```diff
@@ -10,8 +10,7 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
+function getMood(isHappy) {
 
   if (isHappy) {
```

Comment:
There's a pattern in programming to simplify `if .. else ` statements. You can rewrite this block as follows:

```
 if (isHappy) {
    return "I am happy";
  }

return "I am not happy";
```
the pattern is call early return statement, and you can read more here https://medium.com/swlh/return-early-pattern-3d18a41bba8